.env.example

```
@@ -1,56 +1,38 @@
+DATABASE_URL=postgresql://user:password@localhost:5432/volunchain
 
-RATE_LIMIT_AUTH_MAX=10
-RATE_LIMIT_AUTH_WINDOW=3
+# JWT / Security
+JWT_SECRET=your_jwt_secret
 
-RATE_LIMIT_WALLET_MAX=50
-RATE_LIMIT_WALLET_WINDOW=5
+# Supabase
+SUPABASE_URL=https://your-supabase-url.supabase.co
+SUPABASE_ANON_KEY=your_supabase_anon_key
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The SUPABASE_ANON_KEY key should go before the SUPABASE_URL key

(UnorderedKey)